// unabase_api_doc/src/index.js
// const api = "http://35.231.73.122:3000/";
// const api = 'http://localhost:3000/';
// const api = 'http://192.168.0.196:3000/';
// const api = "https://unabase.net/";
// const api = "https://dev.unabase.es/";

const urls = {
  test: "https://test.unabase.es/",
  dev: "https://dev.unabase.es/",
  prod: "https://api.unabase.es/",
  local: "http://localhost:3000/"
};

import tax from "./modules/tax";
import item from "./modules/item";
import user from "./modules/user";
import auth from "./modules/auth";
import business from "./modules/business";
import outcome from "./modules/outcome";
import movement from "./modules/movement";
import currency from "./modules/currency";
import mailer from "./modules/mailer";
import log from "./modules/log";
// import task from './task';

export default function(env) {
  return {
    tax: tax(urls[env]),
    item: item(urls[env]),
    user: user(urls[env]),
    auth: auth(urls[env]),
    business: business(urls[env]),
    outcome: outcome(urls[env]),
    movement: movement(urls[env]),
    currency: currency(urls[env]),
    log: log(urls[env]),
    mailer: mailer(urls[env])
    // task: task(urls[env])
  };
}
