// unabase_api_doc/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});

exports.default = function (env) {
  return {
    tax: (0, _tax2.default)(urls[env]),
    item: (0, _item2.default)(urls[env]),
    user: (0, _user2.default)(urls[env]),
    auth: (0, _auth2.default)(urls[env]),
    business: (0, _business2.default)(urls[env]),
    outcome: (0, _outcome2.default)(urls[env]),
    movement: (0, _movement2.default)(urls[env]),
    currency: (0, _currency2.default)(urls[env]),
    log: (0, _log2.default)(urls[env]),
    mailer: (0, _mailer2.default)(urls[env])
    // task: task(urls[env])
  };
};

var _tax = require("./modules/tax");

var _tax2 = _interopRequireDefault(_tax);

var _item = require("./modules/item");

var _item2 = _interopRequireDefault(_item);

var _user = require("./modules/user");

var _user2 = _interopRequireDefault(_user);

var _auth = require("./modules/auth");

var _auth2 = _interopRequireDefault(_auth);

var _business = require("./modules/business");

var _business2 = _interopRequireDefault(_business);

var _outcome = require("./modules/outcome");

var _outcome2 = _interopRequireDefault(_outcome);

var _movement = require("./modules/movement");

var _movement2 = _interopRequireDefault(_movement);

var _currency = require("./modules/currency");

var _currency2 = _interopRequireDefault(_currency);

var _mailer = require("./modules/mailer");

var _mailer2 = _interopRequireDefault(_mailer);

var _log = require("./modules/log");

var _log2 = _interopRequireDefault(_log);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

// const api = "http://35.231.73.122:3000/";
// const api = 'http://localhost:3000/';
// const api = 'http://192.168.0.196:3000/';
// const api = "https://unabase.net/";
// const api = "https://dev.unabase.es/";

var urls = {
  test: "https://test.unabase.es/",
  dev: "https://dev.unabase.es/",
  prod: "https://api.unabase.es/",
  local: "http://localhost:3000/"
};
// import task from './task';

module.exports = exports.default;
